io.github.JaGoLi.ytdl_gui.yaml

```diff
@@ -0,0 +1,67 @@
+app-id: io.github.JaGoLi.ytdl_gui
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: youtubedl-gui
+
+finish-args:
```

Comment:
I think the way portals are used here is wonky so adding `--filesystem=home` may be the best choice.

Reply:
okay, I'll add the permission